src/main/java/frc/team4909/robot/subsystems/drivetrain/DriveTrainSubsystem.java

```diff
@@ -34,6 +35,18 @@ public DriveTrainSubsystem() {
         bionicDrive = new DifferentialDrive(m_left, m_right);
     }
 
+    public double getElevatorDependentDriveSpeed(double mult){
+        double dependentSpeedMultiplier = RobotConstants.speedMultiplier;
```

Comment:
if this, why the parameter.? I'm not following the logic at first glance... maybe if you add some comments it might seem clearer.